### PR TITLE
Working on travel news

### PR DESCRIPTION
**Pull request by jcreamer898**
This PR provides the UI components for Travel News. 
It also adds a new way to unit test with `npm run ci`. Docs will be added to the wiki for this.

### CODE REVIEW

**Comment by benfalk** [comment]
_No description provided_ :(

**Comment by jcreamer898** [comment]
@benfalk fixed

**Comment by benfalk** [comment]
I would recommend adding more description to your commit messages.  It's okay to say `what` you did.  But a really good thing to include is `why`.  I myself have to remember this, but working with Rowlf more has helped me remember this.

**Comment by jcreamer898** [comment]
Good point. Thanks.

File: spec/unit/components/tabs.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import Tabs from "../../../src/components/tabs/tabs_component";
+import $ from "jquery";
+
+let expect = require("expect.js");
+
+let fixture = `
```
Comment (benfalk) [inline spec/unit/components/tabs.spec.js]: very cool!

Comment (jcreamer898) [inline spec/unit/components/tabs.spec.js]: :sunglasses:

File: src/components/articles/index.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { Component } from "../../core/bane";
 import $clamp from "clamp-js/clamp.js";
+import Tabs from "../tabs/tabs_component";
 
-require("./_articles.scss");
+import "./_articles.scss";
```
Comment (benfalk) [inline src/components/articles/index.js]: what's the difference between `require` and `import` ?

Comment (jcreamer898) [inline src/components/articles/index.js]: In terms of webpack, nothing, just trying to be more consistent.

File: webpack.config.js [hunk 3]
```diff
@@ -16,13 +22,14 @@ module.exports = {
   module: {
     loaders: [{
         test: /\.js$/,
-        exclude: /node_modules\/[^rizzo|flamsteed]/,
+        exclude: /node_modules\/(?!rizzo|flamsteed).*/,
```
Comment (benfalk) [inline webpack.config.js]: is there a better way to exclude these beside a regex?  can you just call them out specifically?

**Comment by benfalk** [comment]
It's mostly greek to me; however it looks good :+1:

File: webpack.config.js (hunk 3 above)
Comment (jcreamer898) [inline webpack.config.js]: Lawd jezzuz I wish.

**Pull request merged**